src/app/fddev/dump.c

```diff
@@ -0,0 +1,88 @@
+#include "fddev.h"
+#include "../../util/net/fd_pcap.h"
+#include <stdio.h>
+
+void
+dump_cmd_args( int      * argc,
+               char * * * argv,
+               args_t   * args ) {
+  char const * out_file = fd_env_strip_cmdline_cstr( argc, argv, "--out-file", NULL, "dump.pcap" );
+  char const * link     = fd_env_strip_cmdline_cstr( argc, argv, "--link",     NULL, "all"       );
```

Comment:
Don't love using `all` here which someone could name a link, better to use NULL?

Reply:
Good point. I switched to using the empty string instead.